README.md

```diff
@@ -9,3 +9,5 @@ This project was created for the creation of my portfolio.
 ## PREREQUISITES
 - Prior knowledge of HTML and CSS.
 
+## LIVE DEMO
+-https://oluyaratosin123.github.io/Portfolio1/
```

Comment:
- [ ] The readme files should have a working link to your portfolio + also the readme should contain more details.